def error(reason): 
    return {'success': False, 'reason': reason}